src/routes/conus/+page.svelte

```diff
@@ -37,31 +39,33 @@
 
 <div class="conus">
 	<Leaflet bind:map height={'600px'}>
-		{#each usstates?.features as feature}
-			<GeoJson
-				geojson={feature}
-				fitBounds={false}
-				fillOpacity={parseInt(feature.properties.STATE) / 100}
-				on:click={stateClick}
-			>
-				<ToolTip>
-					{feature.properties.NAME}
-				</ToolTip>
-			</GeoJson>
-		{/each}
-		{#each counties?.features as county (county.properties.FIPS)}
-			<GeoJson
-				geojson={county}
-				fitBounds={false}
-				fillColor={'red'}
-				fillOpacity={parseInt(county.properties.CNTY_FIPS) / 200}
-				color={'red'}
-				weight="1"
-			>
-				<ToolTip sticky={true}>
-					{county.properties.NAME}
-				</ToolTip>
-			</GeoJson>
-		{/each}
+		<OSMTilelayer>
```

Comment:
should the OSMTileLayer be a sibling of the GeoJSON component, rather than a container?. 

I think the interface could look like this, which makes more sense to me. 
```
<Leaflet>
     <Control>
          <OSM/>
          <MapBox/>
     </Control>
     <GeoJSON />
<Leaflet>
```